### PR TITLE
feat: implement design feedback

### PR DESCRIPTION
**Pull request by joshuanatanielnm**
## Description

Implement some design feedback from @yoniafek , add hafferxh heavy font, update breadcrump link and content based on the cellars

## Changes

List any technical changes.

- [x] update breadcrump link and content
- [x] add hafferxh heavy font for font weight 900

## Screenshots (if appropriate):
image
image
image
image

### CODE REVIEW

**Review by xanderjl — APPROVED** [review]
LGTM 💪

File: src/components/BreadCrumb.tsx [hunk 1]
```diff
@@ -31,13 +35,16 @@ export const BreadCrumb: VFC<CustomBCProps> = ({
...
+          href={landingPageContent ? `/strategies/${id}` : "/"}
           display="flex"
           justifyContent="space-between"
           alignItems="center"
+          gap={3}
         >
-          <ArrowLeftIcon /> Back to Strategies Overview
+          <ArrowLeftIcon />
+          {landingPageContent
+            ? "Back"
```
Comment (codingki) [inline src/components/BreadCrumb.tsx:46]: nice touch

File: src/components/HeadingHeavy.tsx [hunk 2]
```diff
@@ -11,10 +11,10 @@ export const HeadingHeavy: React.FC<Props> = ({
 }) => {
   return (
     <Heading
-      variant="heavy"
       fontSize={{ base: "4xl", md: "7xl" }}
       lineHeight={{ base: "110%", md: "100%" }}
       letterSpacing={{ base: "-1%", md: "-2%" }}
+      fontWeight="900"
```
Comment (codingki) [inline src/components/HeadingHeavy.tsx:17]: just a FYI: we could use `font-black` too here

**Review by codingki — APPROVED** [review]
LGTM

**Pull request merged**